Explorer/Assets/DCL/Character/CharacterMotion/Systems/InterpolateCharacterSystem.cs

```diff
@@ -95,13 +103,26 @@ private void TeleportPlayer(in Entity entity, in CharacterController controller,
             rigidTransform.IsCollidingWithWall = EnumUtils.HasFlag(collisionFlags, CollisionFlags.Sides);
 
             // If we are on a platform we save our local position
-            PlatformSaveLocalPosition.Execute(ref platformComponent, transform.position);
+            PlatformSaveLocalPosition.Execute(ref platformComponent, characterTransform.position);
 
             // In order to detect if we got stuck between 2 slopes we just check if our vertical delta movement is zero when on a slope
             if (rigidTransform.IsOnASteepSlope && Mathf.Abs(deltaMovement.sqrMagnitude) <= ALMOST_ZERO)
                 rigidTransform.IsStuck = true;
             else
                 rigidTransform.IsStuck = false;
         }
+
+        private static Vector3 GravityDelta(float dt, CharacterRigidTransform rigidTransform, CharacterPlatformComponent platformComponent)
```

Comment:
Shall we rename this to `GetGravityDelta`? 🤔 WDYT

Reply:
I was thinking to use it directly in equation and omitting the variable, but the variable is used several time. So I will call it `CalculateGravityDelta`